tests/labs/lie/functional/common.py

```diff
@@ -48,6 +48,11 @@ def get_test_cfg(op_name, dtype, dim, data_shape, module=None):
 
 
 # Sample inputs with the desired types.
+# Input type is one of:
+#   ("tangent", dim)
+#   ("group", module)
+#   ("quat", dim)  # like tangent but normalized
```

Comment:
It might be confusing to say quaternion are like tangent. Do you just mean the values are in a tensor of (..., dim)? .